packages/core/lib/debug/cli.js

```diff
@@ -78,7 +80,12 @@ class CLIDebugger {
           )}.`
         );
       }
-      fetchSpinner.warn(warningStrings.join("  "));
+      // simulate ora's "warn" feature
+      spinners.update({
+        text: `⚠ ${warningStrings.join("  ")}`,
+        color: "yellow",
+        status: "stopped"
+      });
```

Comment:
Shouldn't this take a name, specifying the spinner to update?

Reply:
On that note... could this "warn" feature be a dedicated interface provided by the new package you're adding here?

Reply:
Missing name is indeed a bug. bit odd that the tests didn't catch that... 🤔

Reply:
Re: moving it to the dedicated interface, this is a feature gap between `ora` and `spinnies`, and this is the only place where the `ora` `warn` feature was used. Given that this is a bit of a bodge, I didn't want to encourage its use by adding it to our wrapper.

That said, this would be a fairly straightforward PR for `spinnies`. Perhaps I'll do that and we can just add that to the wrapper when that PR is accepted?

Reply:
Wound up doing that refactoring work, as it made the module much nicer to consume. see #4934 for details on that, as I've split it out from this PR, per @gnidan's request.